src/engine/serverbrowser.cpp

```diff
@@ -253,7 +253,10 @@ ICOMMAND(0, addserver, "siissss", (char *n, int *p, int *r, char *d, char *h, ch
 
 VAR(IDF_PERSIST, searchlan, 0, 0, 1);
 VAR(IDF_PERSIST, maxservpings, 0, 10, 1000);
-VAR(IDF_PERSIST, serverupdateinterval, 0, 10, VAR_MAX);
+VAR(IDF_PERSIST, serverupdateinterval, 0, 15, VAR_MAX);
+VAR(IDF_PERSIST, serverbrowser_show_server_platform_and_branch, 0, 0, 1);
```

Comment:
If this doesn't have a use inside the code, then you should probably be using a persistent alias in `config/menus/servers.cfg` like such:

```
serverbrowser_show_server_platform_and_branch = 0
setpersist serverbrowser_show_server_platform_and_branch 1
```

Either that, or someone should port the `defvar` system from RE2/Tesseract, which lets you define proper variables from scripts.

Reply:
ah, didn't know that works, I'll try it out and thank you very much :D !!